docs/clusters/alpine/software.md

```diff
@@ -57,18 +57,18 @@ Refer to our [Modules System](../../compute/modules.md) page for instructions on
-| [HDF5](https://www.hdfgroup.org/solutions/hdf5/) | 1.10.1, 1.12.1 | HDF5 is a data model, library, and file format for storing and managing data.|
+| [HDF5](https://www.hdfgroup.org/solutions/hdf5/) | 1.10.1, 1.12.1, 1.14.5 | HDF5 is a data model, library, and file format for storing and managing data.|
 | [HOMER](http://homer.ucsd.edu/homer/) | 4.11 | Software for motif discovery and next-gen sequencing analysis.|
 | [IDL](https://www.nv5geospatialsoftware.com/Products/IDL) | 8.7 |IDL software is a scientific programming language used across disciplines to create visualizations out of complex numerical data.|
 | [ImageMagick](https://imagemagick.org/index.php) | 6.9.12 |ImageMagick is a free and open-source software suite for displaying, converting, and editing raster image and vector image files.| 
-| [Intel Advisor](https://www.intel.com/content/www/us/en/developer/tools/oneapi/advisor.html#gs.odmdiw) | 2022.0.0 | Intel Advisor is a design and analysis tool for developing performant code. The tool supports C, C++, Fortran, SYCL, OpenMP, OpenCL code, and Python.|  
+| [Intel Advisor](https://www.intel.com/content/www/us/en/developer/tools/oneapi/advisor.html#gs.odmdiw) | 2022.0.0, 2024.2 | Intel Advisor is a design and analysis tool for developing performant code. The tool supports C, C++, Fortran, SYCL, OpenMP, OpenCL code, and Python.|  
 | Intel cluster tools | 17.3 | Intel cluster tools.| 
-| Intel debugger | 17.3, 2021.5.0 |Intel debugging tools.|
+| Intel debugger | 17.3, 2021.5.0, 2024.2.1 |Intel debugging tools.|
 | [Intel Inspector](https://www.intel.com/content/www/us/en/developer/tools/oneapi/inspector.html#gs.nwibbc) | 2022.0.0 | Intel Inspector is a memory and thread checking and debugging tool to increase the reliability, security, and accuracy of C/C++ and Fortran applications. |
-| [Intel MPI (impi)](https://www.intel.com/content/www/us/en/developer/tools/oneapi/mpi-library.html#gs.nwdbyx) | 2021.4.0, 2021.5.0 |Intel MPI Library is a multifabric message-passing library that implements the open source MPICH specification.| 
-| [Intel oneAPI](https://www.intel.com/content/www/us/en/developer/tools/oneapi/overview.html#gs.nwk0f3) | 2022.1.2 | Intel compilers, libraries, and development tools.|
-| [Intel VTune](https://www.intel.com/content/www/us/en/developer/tools/oneapi/vtune-profiler.html#gs.nwdnjz) | 2022.0.0 | Intel VTune Profiler optimizes application performance, system performance, and system configuration for HPC, cloud, IoT, media, storage, and more.|
-| [Jasper](https://jasper-software.github.io/jasper/) | 4.0.0 (D) | JasPer is a software toolkit for the handling of image data. |
+| [Intel MPI (impi)](https://www.intel.com/content/www/us/en/developer/tools/oneapi/mpi-library.html#gs.nwdbyx) | 2021.4.0, 2021.5.0, 2021.13 (D) |Intel MPI Library is a multifabric message-passing library that implements the open source MPICH specification.| 
```

Comment:
Throughout this table, it appears that we do not specify `(D)`, if it is a module that is dependent on a compiler. I am guessing the reason for this is because you cannot do `module load impi` out the gate, so the version is not default. The default it dependent on the compiler you chose. 

Sticking with that approach, I think you should remove that `(D)` here.